AnalyticsRallyHere/Source/AnalyticsRallyHere/Private/AnalyticsRallyHere.cpp

```diff
@@ -6,12 +6,17 @@
 #include "HttpModule.h"
 #include "Analytics.h"
 
+DEFINE_LOG_CATEGORY(LogAnalyticsRallyHere);
+
 IMPLEMENT_MODULE( FRH_Analytics, AnalyticsRallyHere );
 
 void FRH_Analytics::StartupModule()
 {
 	// Make sure http is loaded so that we can flush events during module shutdown
 	FModuleManager::LoadModuleChecked<FHttpModule>("HTTP");
+
+	// Make sure rally here integration is loaded so we can get the endpoint to emit to
+	FModuleManager::LoadModuleChecked<FHttpModule>("RallyHereIntegration");
```

Comment:
Should the type be changed to the rally here module type?

Reply:
It does not technically matter since we are not using the returned value (and it appears to not do type checking, else it would assert), but good catch.  Will get that adjusted.